cpp/ps0006/cpp_ps0006_solution.cpp

```diff
@@ -4,25 +4,27 @@ using namespace std;
 void Time(int Lnum, int Fnum = 0, int step = 1);
 int main(void)
 {
-
+    // running my valid test cases 2/3
     cout << "the value is:\n";
     Time(5);
     cout << "the value is:\n";
     Time(3, 1);
     cout << "the value is:\n";
     Time(-21, 6, -3);
     cout << "the value is:\n";
-    Time(10, 10, 3);
-    // running invalid test cases, have to give clear error
-    cout << "the value is:\n";
     Time(10, 10, 0);

```

Comment:
- [x] `start=0`, `step=1` as default
- [x] up counter works properly when `step` is positive
- [ ] down counter works properly when `step` is negative